mlrun/model_monitoring/db/tsdb/tdengine/tdengine_connector.py

```diff
@@ -234,9 +234,20 @@ def delete_tsdb_resources(self):
                 drop_statements.append(
                     self.tables[table]._drop_subtable_query(subtable=subtable[0])
                 )
-            self.connection.run(
-                statements=drop_statements, timeout=self._timeout, retries=self._retries
-            )
+            try:
+                self.connection.run(
+                    statements=drop_statements,
+                    timeout=self._timeout,
+                    retries=self._retries,
+                )
+            except Exception as e:
+                logger.warning(
+                    "Failed to delete TDEngine resources, you may need to delete them manually."
+                    "Please note that currently, the available project resources can be found "
+                    "under the following supertables: 'app_results,' 'metrics,' and 'predictions.'",
```

Comment:
Can we be more succinct about it? Something like:
```suggestion
                    "Failed to drop TDEngine tables. You may need to drop them manually. "
                    "These an be found under the following supertables: app_results, "
                    "metrics, and predictions.",
```